main.cpp

```diff
@@ -275,296 +300,281 @@ struct Bakery
-    //get dirty
-    bool getDirty(int numPeople, int numAnimals, float totalFloorSize);
+    bool getDirty(int numPeople, int numAnimals, float hTotalFloorSize);
 };
 
- /*
-Thing 5) vca
-5 properties:
-    1) minimum gain (double)
-    2) maximum gain (double)
-    3) initial gain (double)
-    4) modulation attenuator (float)
-    5) inputs (int)
-3 things it can do:
-    1) set inital gain
-    2) add rhythm to signal
-    3) silence the signal
- */
+void House::provideShelter()
+{
+}
+
+void House::provideRest(float sleepQuality)
+{
+    if (sleepQuality > 3)
+        std::cout << "Conratulations, you managed to get rest!" << std::endl;
+    else
+        std::cout << "Sorry buddy, you'll be sleepy for the rest of the day." << std::endl;
+}
+
+bool House::getDirty(int numPeople, int numAnimals, float hTotalFloorSize)
+{
+    if (hTotalFloorSize / (numPeople+numAnimals) > 20)
+        return true;
+
+    return false;
+}
 
 struct VCA
 {
-    //minimum gain (double)
     double minGain = -100.66;
-    //maximum gain (double)
     double maxGain = 10.5;
-    //initial gain (double)
     double initGain = -20.1;
-    //modulation attenuator (float)
     float modAtt = -56.2f;
-    //inputs (int)
     int inputs = 2;
 
-    //set inital gain
-    float setInitGain(double minGain, double maxGain);
-    //add rhythm to signal
-    void modulateOutput(float modAtt, float modSpeed);
-    //silence the signal
+    double setInitGain(double vMinGain, double vMaxGain);
+    void modulateOutput(float vModAtt, float modSpeed);
     void silenceOutput();
 };
 
-/*
-Thing 6) filter
-5 properties:
-    1) cutoff frequency (float)
-    2) resonance (float)
-    3) filter type (std::string)
-    4) input gain (double)
-    5) overdrive (float)
-3 things it can do:
-    1) cut low frequencies
-    2) cut high frequencies
-    3) self-resonate
- */
+double VCA::setInitGain(double vMinGain, double vMaxGain)
+{
+    std::cout << "Setting init gain. Current minimum is: " << vMinGain << " and maximum is: " << vMaxGain << std::endl;
+
+    return (vMinGain-vMaxGain);
+}
+
+void VCA::modulateOutput(float vModAtt, float modSpeed)
+{
+    std::cout << "Modulating output by: " << vModAtt << " with the frequency of: " << modSpeed << std::endl;
+}
+
+void VCA::silenceOutput()
+{
+}
 
 struct Filter
 {
-    //1) cutoff frequency (float)
     float cutoffFreq = 13200.2f;
-    //2) resonance (float)
     float resonance = 0.77f;
-    //3) filter type (std::string)
     std::string filterType = "Low Pass";
-    //4) input gain (double)
     double inputGain = -5.3;
-    //5) overdrive (float)
     float overdriveAmount = 1.7f;
 
-    //1) cut low frequencies
     void cutLows(float cutoffFreq, float resonance);
-    //2) cut high frequencies
     void cutHighs(float cutoffFreq, float resonance);
-    //3) self-resonate
-    bool selfResonate(float resonance, double inputGain);
+    bool selfResonate(float fResonance, double fInputGain);
```

Comment:
Review our Slack discussion of using member variables in member functions directly. 
Change your argument list so that your function arguments are not duplicates of your member variables.
This can mean changing the function name (meaning the function purpose has changed) 
or it could mean changing the function arguments.
Remember, name things what they are or what they do.
- [ ] revise